### PR TITLE
CNV-18460: virtctl download and installation

### LINKED ISSUE #53361: [enterprise-4.11] Issue in file virt/virt-using-the-cli-tools.adoc
### Which section(s) is the issue in?

Documentation[ OpenShift Container Platform ](https://docs.openshift.com/container-platform/4.11/welcome/index.html)[Virtualization](https://docs.openshift.com/container-platform/4.11/virt/about-virt.html) Using the CLI tools

### What needs fixing?

The user requires the information that it is to required to configure the virtctl binary. This can be done either by setting the environment variable KUBECONFIG like this
```
export KUBECONFIG=/home/dominik/clusters/current/auth/kubeconfig
```
or to login via the "Copy login command" flow.

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.12

Issue: [CNV-18460](https://issues.redhat.com//browse/CNV-18460) [GH#53361](https://github.com/openshift/openshift-docs/issues/53361)

Link to docs preview: [Installing virtctl](https://54135--docspreview.netlify.app/openshift-enterprise/latest/virt/install/virt-installing-virtctl.html#virt-installing-virtctl-client_virt-installing-virtctl)

QE review: Leon Kladnitsky
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/virt-installing-virtctl-client.adoc [hunk 1]
```diff
@@ -1,66 +1,75 @@
...
 +
 [source,terminal]
 ----
 $ echo $PATH
 ----
 
-* For Windows users:
-
-. Unpack and unzip the archive.
+.. Set the `KUBECONFIG` environment variable:
```
Comment (apinnick) [inline modules/virt-installing-virtctl-client.adoc:44]: @dominikholler Please review for [GH#53361](https://github.com/openshift/openshift-docs/issues/53361) .

File: modules/virt-installing-virtctl-client.adoc [hunk 2]
```diff
@@ -1,66 +1,75 @@
...
-// virt/install/virt-enabling-virtctl.adoc
+// * virt/install/virt-installing-virtctl.adoc
 
 :_content-type: PROCEDURE
 [id="virt-installing-virtctl-client_{context}"]
-= Installing the virtctl client
+= Installing the virtctl client on Linux, Windows, and macOS
 
-Extract and install the `virtctl` client after downloading from the appropriate location for your operating system.
+Download and install the `virtctl` client for your operating system.
```
Comment (kelbrown20) [inline modules/virt-installing-virtctl-client.adoc:9]:
```suggestion
You can download and install the `virtctl` client based on your operating system.
```

File: modules/virt-installing-virtctl-client.adoc [hunk 3]
```diff
@@ -1,66 +1,75 @@
...
 
-.Prerequisites
+.Procedure
 
-* You must have downloaded the `virtctl` client.
+. Navigate to *Virtualization > Overview* in the {product-title} web console.
 
-.Procedure
+. Click the *Download virtctl* link on the upper right corner of the page and download the `virtctl` client for your operating system.
```
Comment (kelbrown20) [inline modules/virt-installing-virtctl-client.adoc:15]:
```suggestion
. Click the *Download virtctl* link, which is on the upper right corner of the page, and download the `virtctl` client for your operating system.
```
IBM Style suggests to provide additional descriptive info when describing a location on a page
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

**Review by kelbrown20 — COMMENTED** [review]
Hello! Great work! Just a couple nits, otherwise this looks great

**Pull request merged**